src/index.js

```diff
@@ -126,6 +117,7 @@ class LoadTestClient {
      * @returns Whether the selected participant changed.
      */
     selectStageViewParticipant(selected, previous) {
+        // TODO this needs to select a source instead
```

Comment:
To get the source name for the dominant speaker, you can do this (there is no direct API at the conference level)
```
const videoTrack = this.room.jvbJingleSession?.peerconnection?.getRemoteTracks(selected)?.find(track => track.getType() === 'video');
const sourceName = videoTrack.getSourceName();
```
and use that to update the 'onStageSources' in receiverConstraints.